spec/helpers/alert_notifier_helper_spec.rb

```diff
@@ -0,0 +1,157 @@
+require 'spec_helper'
+
+describe AlertNotifierHelper do
+  describe "#host_and_protocol_for_theme" do
+    include ActionMailerThemer
+
+    context "when the theme is Default" do
+      theme = "default"
+
+      it {
+        expect(helper.host_and_protocol_for_theme(theme))
+          .to eq(
+            host: host(theme),
+            protocol: protocol(theme)
+        )
+      }
```

Comment:
I'm gonna leave this as is because I find it more readable when inputs and outputs in these tests.